msg/Position.msg

```diff
@@ -0,0 +1,5 @@
+std_msgs/Header header
```

Comment:
What is this representing? I don't see the definition of this message type for any topic listed [here](http://robosub.eecs.wsu.edu/wiki/cs/topics_list/start). I would recommend making this clearer as @bkallaher suggested and/or changing the wiki page.